gobblin-core/build.gradle

```diff
@@ -57,6 +57,7 @@ dependencies {
   compile externalDependency.hadoopHdfs
 
   runtime externalDependency.protobuf
+  runtime externalDependency.hadoopAdl
```

Comment:
Yes, this should not be in `gobblin-core`. You could create a module for ADL, but I expect it would be almost empty, otherwise, just make documentation specifying the artifacts that need to be added for running ADL pipeline.